lib/virtfs/core_ext.rb

```diff
@@ -0,0 +1,5 @@
+class String
+  def UnicodeToUtf8
```

Comment:
@roliveri Moving forward, is there a way we can do this without monkey patching the string class?  I'm also concerned that if it's already patched elsewhere, then this code ignores that. Perhaps it should be wrapped in an `unless defined?(UnicodeToUtf8)`